lib/pages/new_private_chat/widget/loading_contact_widget.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:flutter/material.dart';
+
+class LoadingContactWidget extends StatelessWidget {
+  const LoadingContactWidget({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Center(
+      child: Column(
+        children: [
+          const CircularProgressIndicator(),
+          const SizedBox(height: 16.0,),
+          Text("Loading contacts...", 
```

Comment:
We don't have international string for this ? :(

Reply:
sr :))